### PR TITLE
Feature/rm 8637 setting control required true while control enable optional validators false has no effect beyond rendering the required marker

### PR DESCRIPTION
**Pull request by Frazerus**
https://re-motion.atlassian.net/browse/RM-8637

### CODE REVIEW

File: Remotion/ObjectBinding/Web/UI/Controls/BusinessObjectBoundEditableWebControl.cs
Comment (KuJo7) [inline Remotion/ObjectBinding/Web/UI/Controls/BusinessObjectBoundEditableWebControl.cs:1]: remove unused `using Remotion.Web.UI.Controls;`

**Review by KuJo7 — APPROVED** [review]
Looks fine for me :)

**Review by MichaelKetting — CHANGES_REQUESTED** [review]
There's a couple of simplifications we need to do.

**Review by MichaelKetting — APPROVED** [review]
I had some last minute tweaks. Now we just need to split the commits. I think I'll do that quickly.

**Review by MichaelKetting — APPROVED** [review]
Rebase is also done now.

**Pull request merged**